packages/suite/src/actions/wallet/coinjoinClientActions.ts

```diff
@@ -67,14 +74,41 @@ const clientSessionCompleted = (accountKey: string) =>
         },
     } as const);
 
+const clientSessionOwnership = (accountKey: string, roundId: string) =>
+    ({
+        type: COINJOIN.SESSION_OWNERSHIP,
+        payload: {
+            accountKey,
+            roundId,
+        },
+    } as const);
+
+const clientSessionSignTransaction = (accountKey: string, roundId: string) =>
+    ({
+        type: COINJOIN.SESSION_TX_SIGNED,
+        payload: {
+            accountKey,
+            roundId,
+        },
+    } as const);
+
+const clientLog = (payload: string) =>
+    ({
+        type: COINJOIN.CLIENT_LOG,
+        payload,
+    } as const);
+
 export type CoinjoinClientAction =
     | ReturnType<typeof clientEnable>
```

Comment:
What should the `clientEnable` action do? There is no case for `CLIENT_ENABLE` in `coinjoinReducer`.

Reply:
it supposed to set some loader in the ui butiw wasnt implemented at the end, it is there for 3 months already :)